application/classes/Ushahidi/Repository/Post.php

```diff
@@ -526,19 +526,21 @@ protected function setSearchConditions(SearchData $search)
 			}
 		}
 
-		$user = $this->getUser();
 		// If there's no logged in user, or the user isn't admin
 		// restrict our search to make sure we still return SOME results
 		// they are allowed to see
-		if (!$user->id) {
-			$query->where("$table.status", '=', 'published');
-		} elseif (!$this->isUserAdmin($user) and
-				  !$this->acl->hasPermission($user, Permission::MANAGE_POSTS)) {
-			$query
-				->and_where_open()
-				->where("$table.status", '=', 'published')
-				->or_where("$table.user_id", '=', $user->id)
-				->and_where_close();
+		if (!$search->exporter) {
```

Comment:
just a question..  I think if we use the console authorizer we don’t need this, right? (like in the user export)

Reply:
That's a good point, I'll fix that.